packages/utils/upgrade/src/tasks/upgrade/upgrade.ts

```diff
@@ -42,5 +54,5 @@ export const upgrade = async (options: UpgradeOptions) => {
 
   timer.stop();
 
-  logger.info(`Completed in ${timer.elapsedMs}`);
+  logger.info(`Completed in ${f.durationMs(timer.elapsedMs)}`);
```

Comment:
We do have [this](https://github.com/strapi/strapi/blob/07560184b912ddc66dcbfdd750f94067bb6ab05e/packages/core/strapi/src/node/core/timer.ts#L33) if you want to use it

Reply:
I would rather try not to add a Strapi dependency to the upgrade tool